virttest/utils_net.py

```diff
@@ -3737,8 +3737,8 @@ def get_linux_iface_info(iface='', mac=None, session=None):
 
     try:
         ip_output_str = run_func(ip_cmd).strip()
+        LOG.debug(f'Interface info {iface}:\n{ip_output_str}')
```

Comment:
Could you explain the benefit for the change?

Reply:
We met error here, I'd like to check the output string.